### PR TITLE
[Python] Added available options and units to metrics documentation

### PR DESCRIPTION
**Pull request by antonpirker**
This PR adds the available options for configuring metrics in Python and also added the units available in Python.

Preview: https://sentry-docs-git-antonpirker-python-metrics-options-and-units.sentry.dev/platforms/python/metrics/

Related to: https://github.com/getsentry/sentry-docs/issues/9509

### CODE REVIEW

File: docs/platforms/python/common/metrics/index.mdx [hunk 1]
```diff
@@ -12,6 +12,11 @@ Metrics for Python are supported with Sentry Python SDK version `1.40.0` and abo
 
 Sentry metrics help you pinpoint and solve issues that impact user experience and app performance by measuring the data points that are important to you. You can track things like processing time, event size, user signups, and conversion rates, then correlate them back to tracing data in order to get deeper insights and solve issues faster.
 
+## Configure Metrics
+
+You do not need to do anything to enable metrics support, it works out of the box.
```
Comment (vivianyentran) [inline docs/platforms/python/common/metrics/index.mdx:17]:
```suggestion
Metrics is supported out of the box, so you don't need to do anything to enable it. There are some experimental [options](#options) you can use to influence metrics behavior.
```

File: docs/platforms/python/common/metrics/index.mdx [hunk 2]
```diff
@@ -95,3 +100,122 @@ To emit a timer, do the following:
 with sentry_sdk.metrics.timing(key="event_processing_time"):
 	process()
 ```
+
+
+## Options
+
+You do not need to do anything to enable metrics support. There are some options you can set/change to influence the metrics behavior. As Metrics is still in the beta phase, those options are under the `_experimental` option.
```
Comment (vivianyentran) [inline docs/platforms/python/common/metrics/index.mdx:107]:
```suggestion
You can use the following options to set/change the behavior of Metrics. As Metrics is still in the beta phase, these options are under the `_experiments` option.
```

File: docs/platforms/python/common/metrics/index.mdx [hunk 3]
```diff
@@ -95,3 +100,122 @@ To emit a timer, do the following:
...
+		"metrics_summary_sample_rate": 0.5,
+	}
+)
+```
+
+</ConfigKey>
+
+<ConfigKey name="should_summarize_metric">
+
+A callback function hat is called with a metric `key` and the `tags` associated with that key. If the callback function returns `True` a summary for the metrics with the given `key` is generated, if the callback function returns `False` no summary of the metric with the given `key` is generated.
```
Comment (vivianyentran) [inline docs/platforms/python/common/metrics/index.mdx:126]:
```suggestion
A callback function that is called with a metric `key` and the `tags` associated with that key. If the callback function returns `True`, a summary for the metrics with the given `key` is generated. If the callback function returns `False`, no summary of the metric with the given `key` is generated.
```

File: docs/platforms/python/common/metrics/index.mdx [hunk 4]
```diff
@@ -95,3 +100,122 @@ To emit a timer, do the following:
...
+		"should_summarize_metric": should_summarize_metric,
+	}
+)
+```
+
+</ConfigKey>
+
+<ConfigKey name="before_emit_metric">
+
+A callback function that is called before a metric is emmited. If the callback returns `False` the metric is not emitted, if the callback returns `True` the metric is emitted. The given tags can also be updated in the callback function.
```
Comment (vivianyentran) [inline docs/platforms/python/common/metrics/index.mdx:145]:
```suggestion
A callback function that is called before a metric is emmited. If the callback returns `True`, the metric is emitted. If the callback returns `False`, the metric is not emitted. The given tags can also be updated in the callback function.
```

File: docs/platforms/python/common/metrics/index.mdx [hunk 5]
```diff
@@ -95,3 +100,122 @@ To emit a timer, do the following:
...
+		"before_emit_metric": before_emit,
+	}
+)
+```
+
+</ConfigKey>
+
+<ConfigKey name="metric_code_locations">
+
+If `True` the line of code where the metric was emitted will be added to the metric. The code location will be omitted when this option is set to `False`.
```
Comment (vivianyentran) [inline docs/platforms/python/common/metrics/index.mdx:169]:
```suggestion
If `True`, the line of code where the metric was emitted will be added to the metric. If `False`, the code location will be omitted.
```
Would be helpful to add a line saying what the default value is here.

File: docs/platforms/python/common/metrics/index.mdx [hunk 6]
```diff
@@ -95,3 +100,122 @@ To emit a timer, do the following:
...
+- `pebibyte`
+- `exabyte`
+- `exbibyte`
+
+### Fraction Units
+
+- `ratio`
+- `percent`
+
+If you want to explore further, you can find details about supported units in our [event ingestion documentation](https://getsentry.github.io/relay/relay_metrics/enum.MetricUnit.html).
```
Comment (vivianyentran) [inline docs/platforms/python/common/metrics/index.mdx:221]:
```suggestion
For more details about supported units, see our [event ingestion documentation](https://getsentry.github.io/relay/relay_metrics/enum.MetricUnit.html).
```

**Review by vivianyentran — APPROVED** [review]
Added a couple copy edits, but otherwise looks good! Thanks for adding!

File: docs/platforms/python/common/metrics/index.mdx (hunk 4 above)
Comment (antonpirker) [reply docs/platforms/python/common/metrics/index.mdx]: updated

File: docs/platforms/python/common/metrics/index.mdx (hunk 3 above)
Comment (antonpirker) [reply docs/platforms/python/common/metrics/index.mdx]: updated

File: docs/platforms/python/common/metrics/index.mdx (hunk 2 above)
Comment (antonpirker) [reply docs/platforms/python/common/metrics/index.mdx]: updated

File: docs/platforms/python/common/metrics/index.mdx (hunk 1 above)
Comment (antonpirker) [reply docs/platforms/python/common/metrics/index.mdx]: updated

File: docs/platforms/python/common/metrics/index.mdx (hunk 5 above)
Comment (antonpirker) [reply docs/platforms/python/common/metrics/index.mdx:173]: done

File: docs/platforms/python/common/metrics/index.mdx (hunk 6 above)
Comment (antonpirker) [reply docs/platforms/python/common/metrics/index.mdx]: done.

**Pull request merged**